src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -1,7 +1,6 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
```

Comment:
Think we should just change the properties of an object instead of removing the entire thing. Extra work

Reply:
oh yeah but it was in the way when trying to change the addressbook instances so i just used what i did in the tutorials haha